classes/submission/reviewer/form/PKPReviewerReviewStep3Form.php

```diff
@@ -277,6 +278,7 @@ public function saveForLater()
             'recommendation' => (int) $this->getData('recommendation'), // save the recommendation to the review assignment
         ]);
 
+        Hook::call(strtolower_codesafe(get_class($this)) . '::saveForLater', array($this));
```

Comment:
As of PHP8.2, the PHP `strtolower` function is no longer locale-dependent, so we've removed the `strtolower_codesafe` function from our codebase in `stable-3_5_0` and `main`. Use `strtolower` instead. Also, the `[ ... ]` syntax is preferred to `array( ... )`.
```suggestion
        Hook::call(strtolower(get_class($this)) . '::saveForLater', [$this]);
```

Reply:
I did replace the commit with the suggested changes from you.